README.md

```diff
@@ -73,33 +73,41 @@ See [@typescript-eslint/parser's README.md](../parser/README.md) for more inform
-| [`ts-immutable/no-object-mutation`](./docs/rules/no-object-mutation.md)   | Disallow mutating objects                                                 | :heavy_check_mark: |          | :thought_balloon: |
-| [`ts-immutable/no-method-signature`](./docs/rules/no-method-signature.md) | Prefer property signatures with readonly modifiers over method signatures | :heavy_check_mark: |          |                   |
-| [`ts-immutable/no-delete`](./docs/rules/no-delete.md)                     | Disallow delete expressions                                               | :heavy_check_mark: |          |                   |
+| Name                                                                      | Description                                                               | <span title="Recommended">:see_no_evil:</span> | <span title="Functional Lite">:hear_no_evil:</span> | <span title="Functional">:speak_no_evil:</span> | :wrench: | :blue_heart: |
+| ------------------------------------------------------------------------- | ------------------------------------------------------------------------- | ---------------------------------------------- | --------------------------------------------------- | ----------------------------------------------- | -------- | ------------ |
+| [`ts-immutable/readonly-keyword`](./docs/rules/readonly-keyword.md)       | Enforce readonly modifiers are used where possible                        | :heavy_check_mark:                             | :heavy_check_mark:                                  | :heavy_check_mark:                              | :wrench: |              |
+| [`ts-immutable/readonly-array`](./docs/rules/readonly-array.md)           | Prefer readonly array over mutable arrays                                 | :heavy_check_mark:                             | :heavy_check_mark:                                  | :heavy_check_mark:                              | :wrench: |              |
+| [`ts-immutable/no-let`](./docs/rules/no-let.md)                           | Disallow mutable variables                                                | :heavy_check_mark:                             | :heavy_check_mark:                                  | :heavy_check_mark:                              | :wrench: |              |
+| [`ts-immutable/no-array-mutation`](./docs/rules/no-array-mutation.md)     | Disallow mutating arrays                                                  | :heavy_check_mark:                             | :heavy_check_mark:                                  | :heavy_check_mark:                              |          | :blue_heart: |
+| [`ts-immutable/no-object-mutation`](./docs/rules/no-object-mutation.md)   | Disallow mutating objects                                                 | :heavy_check_mark:                             | :heavy_check_mark:                                  | :heavy_check_mark:                              |          | :blue_heart: |
```

Comment:
`no-object-mutation` doesn't need type information to work but it will use it if it is available (rule works without it, works better with it).
Should the last column be empty or use another symbol to state the type information is optional?

Reply:
I think another symbol would be nice. I got the initial symbols from the [typescript-eslit](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules) plugin rules. They use :thought_balloon: = requires type information (not sure how that symbol is related to type info :-)).

Perhaps we can use 
:thought_balloon: = requires type information
:blue_heart: = works better with type information

Reply:
Sounds good ☺️